.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libsass"]
 path = libsass
-url = git://github.com/sass/libsass.git
+url = git@github.com:/sass/libsass.git
```

Comment:
would prefer https here so it's clonable by others without needing ssh keys (and in CI)

Reply:
```suggestion
url = https://sass/libsass.git
```